components/AvalancheDangerTable.tsx

```diff
@@ -50,26 +48,31 @@ export const AvalancheDangerTable: React.FunctionComponent<AvalancheDangerTableP
 
   return (
     <VStack space={12} alignItems="stretch">
-      <Body>{utcDateToLocalDateString(date)}</Body>
+      {date && <BodySmBlack>{utcDateToLocalDateString(date)}</BodySmBlack>}
       <View height={height} width="100%">
         {/* This view contains 3 layers stacked over each other: the background bars in gray, the avalanche pyramid, and the text labels and icons */}
-        <VStack width="100%" height="100%" position="absolute" justifyContent="space-evenly" alignItems="stretch" space={3} paddingTop={paddingTop} zIndex={10}>
+        <VStack width="100%" height="100%" position="absolute" justifyContent="space-evenly" alignItems="stretch" space={4} zIndex={10}>
           <View bg="gray.100" flex={1} />
           <View bg="gray.100" flex={1} />
           <View bg="gray.100" flex={1} />
         </VStack>
         <View width="100%" height="100%" position="absolute" zIndex={20}>
-          <AvalancheDangerPyramid forecast={danger} height="100%" style={{marginLeft: marginLeft}} />
+          <AvalancheDangerTriangle forecast={danger} height="100%" style={{marginLeft: marginLeft}} />
         </View>
-        <VStack width="100%" height="100%" position="absolute" justifyContent="space-evenly" alignItems="stretch" space={3} paddingTop={paddingTop} zIndex={30}>
+        <VStack width="100%" height="100%" position="absolute" justifyContent="space-evenly" alignItems="stretch" space={4} zIndex={30}>
           {(['upper', 'middle', 'lower'] as const).map((layer, index) => (
-            <HStack flex={1} justifyContent="space-between" key={index}>
-              <View my={4} px={1} justifyContent="center">
-                <Caption1>{(elevation_band_names[layer] ?? defaultElevationBands[layer]).replace('<br>', '\n')}</Caption1>
-              </View>
+            <HStack flex={1} justifyContent="space-between" flexDirection={elevation_band_names ? 'row' : 'row-reverse'} key={index} paddingHorizontal={4}>
+              {elevation_band_names && (
+                <View my={4} px={1} justifyContent="center">
+                  <Caption1>{(elevation_band_names[layer] ?? defaultElevationBands[layer]).replace('<br>', '\n')}</Caption1>
+                </View>
+              )}
```

Comment:
It looks like this label is meant to be in a white box with a small corner radius (probably 2px?) and a 1px border (color `#F3F5F6`, not sure if that's a predefined color or an improvisation from the design side 😄)

<img width="378" alt="image" src="https://github.com/NWACus/avy/assets/101196/19bdce91-b38a-4108-b4fe-6200d7200587">

Reply:
Oh and I guess it's meant to be `<Caption1Semibold>` for the first line

Reply:
Hm - their designs have always had the white box in them and we've never actually made that happen. Maybe let's confirm with them in a follow-up if that's really the intent? I figured if it were the intent we would have heard about it in design review sometime in the last year :D 

(attached a design iteration from early last season) 

![Screenshot from 2023-10-09 07-04-26](https://github.com/NWACus/avy/assets/7328370/42f31562-c309-4315-8ab9-cde5a44859de)